controllers/utils/peer_ref.go

```diff
@@ -93,43 +93,23 @@ func GetClusterID(ctx context.Context, client client.Client, clusterName string)
 }
 
 func getPeerRefType(ctx context.Context, c client.Client, peerRef multiclusterv1alpha1.PeerRef, isManagedCluster bool) (PeerRefType, error) {
+	operatorNamespace := GetEnv("POD_NAMESPACE")
 	if isManagedCluster {
-		operatorNamespace := GetEnv("POD_NAMESPACE")
-		cm, err := GetODFInfoConfigMap(ctx, c, operatorNamespace)
-		if err != nil {
-			return PeerRefTypeUnknown, fmt.Errorf("failed to get ODF Info ConfigMap for namespace %s: %w", peerRef.StorageClusterRef.Namespace, err)
-		}
-		var odfInfo ocsv1alpha1.OdfInfoData
-		for key, value := range cm.Data {
-
-			err := yaml.Unmarshal([]byte(value), &odfInfo)
-			if err != nil {
-				return PeerRefTypeUnknown, fmt.Errorf("failed to unmarshal ODF info data for key %s: %w", key, err)
-			}
-
-			for _, client := range odfInfo.Clients {
-				if client.Name == peerRef.StorageClusterRef.Name {
-					return PeerRefTypeStorageClient, nil
-				}
-			}
+		operatorNamespace = GetEnv("HUB_OPERATOR_NAMESPACE")
+	}
```

Comment:
Instead of calculating the namespace here, why not calculate it before we call the IsStorageClientType and remove the need for isManagedCluster?

Reply:
We won't need a GetEnv when we are running on the hub. we can directly pass the currentNamespace that we calculate